inc/dps/registration.h

```diff
@@ -194,6 +202,10 @@ DPS_Status DPS_Registration_LinkTo(DPS_Node* node, DPS_RegistrationList* regs, D
  */
 DPS_Status DPS_Registration_LinkToSyn(DPS_Node* node, DPS_RegistrationList* regs, DPS_NodeAddress* addr);
 
+/** @} */ // end of registration group
+
+/** @} */ // end of services group
+
```

Comment:
All of the documentation here is placed in both the registration group and the services group. Is that what you intended?

Reply:
Yes, registration is a sub-group of services.